app/src/main/java/com/lastaoutdoor/lasta/ui/screen/settings/SettingsScreen.kt

```diff
@@ -56,11 +83,12 @@ fun SettingsScreen(
             updatePrefActivity = updatePrefActivity,
             updateClimbingLevel = updateClimbingLevel,
             updateHikingLevel = updateHikingLevel,
-            updateBikingLevel = updateBikingLevel)
+            updateBikingLevel = updateBikingLevel,
+            setUpOrSetting = false)
 
         Spacer(modifier = Modifier.height(24.dp))
```

Comment:
yep in this case the space is so small that not really necessary because it won't change something to the human eye unless the phone is like 3x smaller but why not for good measures